### ISSUE TITLE
Batteries 2.4 - Release announcement

### ISSUE THREAD

**Issue by Fourchaux** [ocaml-batteries-team/batteries-included#660]
Sur la page d'accueil du site http://batteries.forge.ocamlcore.org/
(même si le nombre de comment est de 0 en 3 ans - oups -) une petite ligne 'Batteries 2.4 released' (ainsi d'ailleurs, qu'une ligne 'Batteries 2.3 released' ... soyons fous !!! )  remplirait un peu la rubrique 'Latest News' ...

**Comment by Blaisorblade** [comment]
I agree. After looking at http://batteries.forge.ocamlcore.org/ I almost asked "Is Batteries dead?". While I'm not sure Ocaml Forge has a point nowadays (and seems rather [dead](https://forge.ocamlcore.org/)), that's still the first Google result for `ocaml batteries`.

I'd recommend declaring that website deprecated and adding a link to whatever the current homepage is.

**Comment by gasche** [comment]
Thanks for the report and the ping. As a stopgap measure, I published a news on the new release 2.4.0. I am uncomfortable with deprecating the forge as we don't have another webpage other than the github page (and its README). I think the content of the Forge frontpage is actually decent, but we may need to clarify the fact that development happens mostly on github nowadays.

**Comment by Blaisorblade** [comment]
:+1:, but https://forge.ocamlcore.org/forum/forum.php?forum_id=928 gives me "Permission denied".

Also, nice to see you here @gasche :smile: 

> I think the content of the Forge frontpage is actually decent.

Well, if up-to-date it might not be a problem. Also, right now, the README does functions like a homepage for users (and doesn't point to the Forge webpage), so there seems to be some duplication (where I prefer the README).

Also, I'm not objecting to the content of [the Batteries Forge page](http://batteries.forge.ocamlcore.org/). But the [Forge itself](https://forge.ocamlcore.org/) feels dead to me, because 
- it seems not very used (only 7 news in 2015);
- it looks like SourceForge for Ocaml, not like GitHub for Ocaml, which nowadays is uncool (because it gives a worse user experience).

That's just my first superficial impression as an outsider and Ocaml "newbie", so please take for what it's worth (not so much); and I didn't look very deeply into the website.

**Comment by gasche** [comment]
The forge is being deprecated in 2017, so we have to move the HTML to static github pages -- no big deal. I'm renaming this issue to keep track of this.
